xbmc/interfaces/legacy/Addon.h

```diff
@@ -100,10 +100,157 @@ namespace XBMCAddon
 #ifdef DOXYGEN_SHOULD_USE_THIS
       ///
       /// \ingroup python_xbmcaddon
-      /// @brief \python_func{ xbmcaddon.Addon([id]).getSettings() }
+      /// @brief \python_func{ xbmcaddon.Addon([id]).supportsInstanceSettings() }
+      /// Returns true if the requested add-on supports several instances.
+      ///
+      /// @return Returns true if supported, false otherwise
+      ///
+      ///
+      ///-----------------------------------------------------------------------
+      /// @python_v21 New function added.
+      ///
+      ///
+      /// **Example:**
+      /// ~~~~~~~~~~~~~{.py}
+      /// ..
+      /// supportsInstances = self.Addon('pvr.iptvsimple').supportsInstanceSettings()
+      /// ..
+      /// ~~~~~~~~~~~~~
+      ///
+      supportsInstanceSettings();
+#else
+      bool supportsInstanceSettings();
+#endif
+
+#ifdef DOXYGEN_SHOULD_USE_THIS
+      ///
+      /// \ingroup python_xbmcaddon
+      /// @brief \python_func{ xbmcaddon.Addon([id]).getKnownInstanceIds() }
+      /// Returns the list of known available instance ids.
+      ///
+      /// @return List of used add-on instances
+      ///
+      ///
+      ///-----------------------------------------------------------------------
+      /// @python_v21 New function added.
+      ///
+      ///
+      /// **Example:**
+      /// ~~~~~~~~~~~~~{.py}
+      /// ..
+      /// usedInstances = self.Addon('pvr.iptvsimple').getKnownInstanceIds()
+      /// ..
+      /// ~~~~~~~~~~~~~
+      ///
+      getKnownInstanceIds();
```

Comment:
I think known is pretty much implicit. Maybe rename to getInstanceIds()?

Reply:
Have changed it to `getInstanceIds`, outside of python I leaved still as `GetInstanceIds()`, maybe I change in another request to have all with same name a again.